person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
```

Comment:
- [ ] Kindly add `@id` inside constructor and give it random number, eg.:
`@id = Random.rand(1..1000)`

- [ ] I strongly recommend starting the page from getters and setters to follow best practices: 

```ruby
class Person
  attr_accessor :name, :age 
  attr_reader :id 
 
  def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```